web/src/engine/main/osk/banner.ts

```diff
@@ -164,9 +174,26 @@ namespace com.keyman.osk {
     }
   }
 
+  // TODO:  finalize + document
+  interface OptionFormatSpec {
+    minWidth?: number;
+    paddingWidth: number,
+    emSize: number,
+    styleForFont: CSSStyleDeclaration
+
+    collapsedWidth?: number
+  }
+
```

Comment:
Obviously a significant TODO; see other comment in this grouping for example uses.